rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1049,7 +1051,19 @@ def test_add_blockchain_accounts_with_tags_and_label_and_querying_them(rotkehlch
         'blockchainsaccountsresource',
         blockchain='ETH',
     ), json={'accounts': accounts_data})
-    assert_proper_response(response)
+    result = assert_proper_response_with_result(response)
+    for account_data_entry in accounts_data:
+        for result_entry in result['result']:
+            if account_data_entry['address'] == result_entry['address']:
+                if 'label' in account_data_entry:
+                    assert result_entry['label'] == account_data_entry['label']
+                if 'tags' in account_data_entry:
+                    assert set(result_entry['tags']) == set(account_data_entry['tags'])
+                break  # break means that we found the account data entry
+        else:  # else means that `break` has not happened
+            raise AssertionError(f'Not found entry {account_data_entry} in returned result')
```

Comment:
No, I can't since the order of the accounts in the list might be different and the order of the tags in the tags list of a particular account can also be different

Reply:
Couldn't you make a set so the order doesn't matter and iterate the result and make a set out of the field that is needed in the response? Also are you sure that the order in the response is different each time?